azure-pipelines.yml

```diff
@@ -259,6 +259,33 @@ jobs:
     artifact: dist-partial-top_earlgrey_nexysvideo
     displayName: 'Upload partial distribution artifacts'
 
+- job: "riscv_compliance_tests"
+  displayName: "Execute RISC-V compliance tests"
+  pool: "Default"
+  dependsOn:
+    - top_earlgrey_verilator
+  steps:
+  - bash: |
+      sudo apt-get install -y python3 python3-pip build-essential srecord python3-setuptools zlib1g-dev libusb-1.0 libftdi1-dev libftdi1-2 libssl-dev \
```

Comment:
eventually i'm going to include fpga riscv compliance testing, so maybe that means i would need to build the spiflash tools? if they eventually get built earlier and archived, i'll remove some of the dependencies here.